src/components/shared/Markdown.tsx

```diff
@@ -1,8 +1,10 @@
+import { ExclamationTriangleIcon } from "@radix-ui/react-icons";
+import { Callout } from "@radix-ui/themes";
 import { omit } from "lodash";
 import { type MarkedOptions, parse, parseInline } from "marked";
 import type * as React from "react";
+import { t } from "ttag";
```

Comment:
issue: ttag doesn't exist in this project anymore

```suggestion
import { t } from "@transifex/native";
```